api/sheets/get_variable.md

```diff
@@ -1,111 +1,122 @@
 ---
 layout: default
-title: Statistical Variable Value
+title: Statistical Variable Values
 nav_order: 3
 parent: Google Sheets
 grand_parent: API
 ---
 
-# Get the value of a statistical variable at a given place and time
+# Getting the Value of a Statistical Variable at a Given Place and Time
 
-Given a list of [DCIDs](/glossary.html) representing [Place](https://datacommons.org/browser/Place) objects, a [StatisticalVariable](https://datacommons.org/browser/StatisticalVariable), and optionally a date, get the measurements of the specified variable in the specified places with a date (if specified).
+The`=DCGET(dcids, variable, date)` formula returns the measurements of a specified [statistical variable](https://docs.datacommons.org/glossary.html#variable) at a given place and optional time based on a list of parent [Place](https://datacommons.org/browser/Place) [DCIDs](https://docs.datacommons.org/glossary.html). A complete list of variables can be found in the [graph browser](https://datacommons.org/browser/StatisticalVariable).
```

Comment:
add direct link to dcid
```suggestion
The`=DCGET(dcids, variable, date)` formula returns the measurements of a specified [statistical variable](https://docs.datacommons.org/glossary.html#variable) at a given place and optional time based on a list of parent [Place](https://datacommons.org/browser/Place) [DCIDs](https://docs.datacommons.org/glossary.html#dcid). A complete list of variables can be found in the [graph browser](https://datacommons.org/browser/StatisticalVariable).
```

Reply:
Also, do we want to also link to the stat var explorer instead of the graph browser for a complete list of variables?

Reply:
yes we do - updating the links.